rconweb/api/history.py

```diff
@@ -19,9 +20,10 @@
 from .auth import login_required, api_response
 from .utils import _get_data
 from rcon.discord import send_to_discord_audit
-
+from rcon.settings import SERVER_INFO
 
 logger = logging.getLogger("rconweb")
+ctl = RecordedRcon(SERVER_INFO)
```

Comment:
It would be better not to create another one. Given that you have N number of workers in the backend that would create N more connection at import time.

It was a crappy design choice that I did very early in the process to create those connection in the global scope (import time), but we should try not to make it worse :)

Import the one from the `.api` module

Reply:
> Import the one from the .api module
 where is it ? I didn t find where the rcon is defined in the api module

Reply:
not sure if it was what you asked but see last commit